_docs/samples/consul-integration.md

```diff
@@ -33,9 +33,7 @@ Note that Istio pilot agent is running inside each app container so as to coordi
 1. First step is to generate a configuration file which will be used by `istioctl` and Istio Discovery
 
     ```bash
-    kubectl config set-cluster local --server=http://172.28.0.13:8080
-    kubectl config set-context local --cluster=local
-    kubectl config use-context local
+    istioctl context-create --api-server http://172.28.0.13:8080
```

Comment:
We also need to tweak the "Before you begin" section, and remove the kubectl part there. Ideally, if we are bringing up the API server as part of docker compose, we don't need any reference to the kube installation instructions right?

Reply:
you are right, I missed removing kubectl - updated